### ISSUE TITLE
Links wrapped in angle brackets disappear when fetching thread contents from the command line

### ISSUE THREAD

**Issue by Oblomov** [ihabunek/toot#363]
[Example post](https://sociale.network/@oblomov/110395233362332095). Trying to fetch this (`toot thread <id>`) leads to:

```
Challenging question : was Google really responsible for killing off #RSS by phasing out #GoogleReader?
Those that follow me know that at least I for one put at least some weight on @mozilla <http://wok.oblomov.eu/tecnologia/opera-requiem-3/ and https://sociale.network/@oblomov/109902001164978888>, but I still disagree with @eniko's view.
Several in the comments there have already mentioned that alternatives to #GoogleReader _do_ exist, but two comments in particular hit (IMO) the nail in the head:

1/n
```

versus the expected 
```
Challenging question <https://peoplemaking.games/@eniko/110394794029226907>: was Google really responsible for killing off #RSS by phasing out #GoogleReader?
Those that follow me know that at least I for one put at least some weight on @mozilla <http://wok.oblomov.eu/tecnologia/opera-requiem-3/ and https://sociale.network/@oblomov/109902001164978888>, but  I still disagree with @eniko's view.
Several in the comments there have already mentioned that alternatives to #GoogleReader _do_ exist, but two comments in particular hit (IMO) the nail in the head:

1/n
```

**Comment by danschwarz** [comment]
Bug verified against master - and, strangely, also verified against my PR #359  which uses an entirely different method of rendering status text.  :thinking:

**Comment by strk** [comment]
How does a thread id look like ? I'm getting `Unknown error` when using:

> toot thread https://goto.77nn.it/@77nn/statuses/01HGARTFD523B6DV8FJPS6XBXB

**Comment by ihabunek** [comment]
This should be fixed since 0.40.0 since the `colorize` function which caused this is no longer used.

**Comment by ihabunek** [comment]
> How does a thread id look like ? I'm getting `Unknown error` when using:
> 
> > toot thread https://goto.77nn.it/@77nn/statuses/01HGARTFD523B6DV8FJPS6XBXB

You can't use URLs, you need to use the **status** ID for which you want to show the thread.
